frontend/src/components/pages/TeamDashboard/index.jsx

```diff
@@ -65,12 +71,9 @@ function DashboardUnconnected({
 
 function mapStateToProps(state) {
   return {
-    teams: state.apiEntities.teams || {},
-    authUser: state.App.authUser || {},
-
-    activityLogDayCounts: Object.values(
-      state.apiEntities.activityLogDayCounts || {}
-    ),
+    teams: state.apiEntities.teams,
+    authUser: state.App.authUser,
+    activityLogDayCounts: state.apiEntities.activityLogDayCounts,
```

Comment:
Surely this breaks the activitylogDayCounts somewhere? We were expecting a list containing the values but we are now passing an object through. this would mean that wherever it is used, is not getting the data in the format it was before.

My recommendation here is to make any change on line 26. Ensure that it retains the format we are expecting else, stuff is going to break